requirements.txt

```diff
@@ -3,3 +3,7 @@ boto3 >= 1.7.13
 crc32c
 puremagic==1.4
 requests >= 2.18.4, < 3
+pyjwt >= 1.5.3, < 2
+cryptography >= 2.3, < 3
+furl >= 1.2.1
+hca
```

Comment:
is there a specific reason to not pin to a version of `hca`?

Reply:
1. The `hca` dependency should not be included here, because `hca` depends on `dcplib`, so this would create a circular dependency.
2. Libraries should not pin to specific versions of their dependencies unless there is a very specific reason. This is because libraries installed alongside each other may each have the same dependency; the more of them pin the dependency, the less likely the pin is to be satisfied. Instead, libraries should declare a dependency on a major version range, like the ones above on pyjwt and cryptography. Services using the libraries can still pin a specific version of the dependency in their `pip freeze` requirements.

Reply:
removed `hca` dependency. It was add because because I saw it was used here https://github.com/HumanCellAtlas/dcplib/blob/dc2cfd154340d59cf35993f63288333f24ea5236/dcplib/etl/__init__.py#L23